discovery-provider/src/queries/get_app_name_metrics.py

```diff
@@ -0,0 +1,40 @@
+import logging
+from sqlalchemy import desc
+
+from src.models import AppNameMetrics
+from src.utils import db_session
+
+logger = logging.getLogger(__name__)
+
+
+def get_app_name_metrics(app_name, args):
+    """
+    Returns the usage metrics for a specified app_name
+
+    Parameters:
```

Comment:
we should try and be consistent across the db with how we do docstrings. do you like this format more than what I did? i think my thing was "technically" correct for pydoc, but i'm not opposed to this

https://stackoverflow.com/questions/34331088/how-to-comment-parameters-for-pydoc

wdyt?